week-2/13-order/readme.md

```diff
@@ -0,0 +1,13 @@
+# Item order
+
+The `flex-order` property is used to move a *specific* item to a different position in the list.
+
+> **Remember:** if you forgot how to use the `flex-order` property, you can always:
+> 1. Google the word "flexbox"
+> 2. Click on the link from "CSS-tricks"
+> ![Screenshot of how to google flexbox](/images/13-googling-flexbox.png)
+
+Exercise:
```

Comment:
I've been excessively descriptive when saying what they need to do. Maybe that's not a good idea. But I'd use something like this:

> In this exercise, you'll use `order` to move the items in the list. Follow the steps below to complete this exercise.